app/pb_kits/playbook/pb_currency/_currency.jsx

```diff
@@ -1,72 +1,75 @@
 /* @flow */
 
-import React from 'react'
-import classnames from 'classnames'
+import React from "react";
+import classnames from "classnames";
 
-import {
-  Body,
-  Caption,
-  Title,
-} from '../'
+import { Body, Caption, Title } from "../";
 
 type CurrencyProps = {
-  align?: 'left' | 'center' | 'right',
+  align?: "left" | "center" | "right",
   className: String,
-  currencySymbol: '$',
+  currencySymbol: "$",
   label?: String,
-  separator?: '.' | ',',
-  size?: 'sm' | 'lg',
+  separator?: "." | ",",
+  size?: "sm" | "lg",
   value?: String,
-  unit?: '00',
-}
+  unit?: "00"
+};
 
-const kitCSS = ({align='left'}: CurrencyProps) => {
-  let css = 'pb_currency_kit'
-  css += `_${align}`
-  return css
-}
+const kitCSS = ({ align = "left" }: CurrencyProps) => {
```

Comment:
This [props utility file](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/utilities/props.js#L48) was introduced a few weeks ago with a handful of useful helpers that you may be able to leverage here to dry up a bit the code.

The [pb_toggle kit](https://github.com/powerhome/playbook/blob/89d83abe394e9924787bf079dfef4eeff0fb2987/app/pb_kits/playbook/pb_toggle/_toggle.jsx#L45) makes use of these helpers and should be a good reference.